src/core/abstract/MCMessageConstants.h

```diff
@@ -179,7 +179,8 @@ namespace mailcore {
         IMAPSearchKindSubject,
         IMAPSearchKindContent,
         IMAPSearchKindBody,
-        IMAPSearchKindUIDs,
+		IMAPSearchKindUIDs,
```

Comment:
Could you just fix indentation here from Tabs to space?
And then, I'll merge it.